### PR TITLE
separation of news on category

### PR DESCRIPTION
**Pull request by donni106**
feat(multiple news sections): dynamic news on home screen

- refactored news on home screen
- removed hard coded news section and texts
- added map over multiple news categories to
  create multiple news sections
- `categoriesNews` array needs to be added on the
  main server in `sections` with objects containing
  a `categoryId` to filter, `categoryTitle` to show as
  headlines, `categoryTitleDetail` to show in header
  on details screens and `categoryButton` to show as
  button text

SVA-32

feat(news filter): consider category for data providers

- when filtered news on category, the data provider selection
  on news index also needs to take care of that
- added parameter for category id for data provider query
  in news items
- added TODOs for parts, where the main server is not ready yet
  - ikuseiGmbH/smart-village-app-mainserver#158
    needs to be merge and tested before

SVA-32

### CODE REVIEW

File: src/queries/newsItems.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import gql from 'graphql-tag';
 
 export const GET_NEWS_ITEMS = gql`
-  query NewsItems($limit: Int, $offset: Int) {
+  query NewsItems($limit: Int, $offset: Int, $categoryId: ID) {
```
Comment (donni106) [inline src/queries/newsItems.js:4]: in den queries muss vllt. `$categoryId: String` statt `$categoryId: ID`. wird sich zeigen, wenn server ready ist

File: src/screens/HomeScreen.js [hunk 2]
```diff
@@ -115,76 +120,89 @@ export const HomeScreen = ({ navigation }) => {
...
+                            }
+                          />
+                        </Wrapper>
+                      </View>
+                    );
+                  }}
+                </Query>
+              </Fragment>
+            )
+          )}
```
Comment (SoleCincis) [inline src/screens/HomeScreen.js:206]: How is this result on the screen connected with this PR ? Screenshot 2020-11-09 at 13 00 28

Comment (donni106) [reply src/screens/HomeScreen.js:206]: this screenshot shows the separation of newsitems into two category blocks "nachrichten" and "informationen". before there were all newsitems in one block. now there are some newsitems in the first and others in the second.

File: src/queries/newsItems.js (hunk 1 above)
Comment (donni106) [reply src/queries/newsItems.js:4]: überall nun `$categoryId: ID`

File: src/screens/HomeScreen.js [hunk 3]
```diff
@@ -90,15 +95,15 @@ export const HomeScreen = ({ navigation }) => {
         rootRouteName: 'EventRecords'
```
[the remark this answers is not in the document]
Comment (donni106) [reply src/screens/HomeScreen.js:95]: i don't know exactly what you mean.
the global settings comes from the server initially with starting the app. at the top of this screen we read the data from the global settings. the global settings should be accessible all over the app. that's why they are provided per context.

**Pull request merged**